udc/src/test/java/io/crate/udc/ping/PingTaskTest.java

```diff
@@ -146,11 +154,68 @@ public void testSuccessfulPingTaskRun() throws Exception {
             assertThat(map.get("crate_version"), is(notNullValue()));
             assertThat(map, hasKey("java_version"));
             assertThat(map.get("java_version"), is(notNullValue()));
+            assertThat(map, hasKey("license_expiry_date"));
+            assertThat(map.get("license_expiry_date"), is(PingTask.getLicenseExpiryDate(LICENSE)));
+            assertThat(map, hasKey("license_issued_to"));
+            assertThat(map.get("license_issued_to"), is(PingTask.getLicenseIssuedTo(LICENSE)));
+        }
+    }
+
+    @Test
+    public void testSuccessfulPingTaskRunWhenLicenseIsNull() throws Exception {
+        when(licenseService.currentLicense()).thenReturn(null);
+
+        testServer = new HttpTestServer(18080, false);
+        testServer.run();
+
+        PingTask task = createPingTask("http://localhost:18080/", Settings.EMPTY);
+        task.run();
+        assertThat(testServer.responses.size(), is(1));
+        task.run();
+        assertThat(testServer.responses.size(), is(2));
+        for (long i = 0; i < testServer.responses.size(); i++) {
+            String json = testServer.responses.get((int) i);
+            Map<String, String> map = new HashMap<>();
+            ObjectMapper mapper = new ObjectMapper();
+            try {
+                //convert JSON string to Map
+                map = mapper.readValue(json,
+                    new TypeReference<HashMap<String, String>>() {});
+
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+
+            assertThat(map, hasKey("kernel"));
+            assertThat(map.get("kernel"), is(notNullValue()));
+            assertThat(map, hasKey("cluster_id"));
+            assertThat(map.get("cluster_id"), is(notNullValue()));
```

Comment:
These assertions seem to be irrelevant for the test? Or what exactly are we testing here?